src/InvestJob.jsx

```diff
@@ -12,131 +12,126 @@ import Navbar from 'react-bootstrap/Navbar';
 import Nav from 'react-bootstrap/Nav';
 import Spinner from 'react-bootstrap/Spinner';
 
-import { HomeTab } from './components/HomeTab';
-import { SetupTab } from './components/SetupTab';
-import { LogTab } from './components/LogTab';
-import { ResourcesTab } from './components/ResourcesTab';
-import { LoadButton } from './components/LoadButton';
+import HomeTab from './components/HomeTab';
+import SetupTab from './components/SetupTab';
+import LogTab from './components/LogTab';
+import ResourcesTab from './components/ResourcesTab';
+import LoadButton from './components/LoadButton';
 import { SettingsModal } from './components/SettingsModal';
-import { getSpec, fetchDatastackFromFile,
-         writeParametersToFile } from './server_requests';
-import { argsDictFromObject, findMostRecentLogfile,
-         cleanupDir } from './utils';
+import {
+  getSpec, fetchDatastackFromFile, writeParametersToFile
+} from './server_requests';
+import {
+  findMostRecentLogfile, cleanupDir
+} from './utils';
 import { fileRegistry } from './constants';
-import { getLogger } from './logger'
-const logger = getLogger(__filename.split('/').slice(-1)[0])
+import { getLogger } from './logger';

```

Comment:
What do the brackets do here? I've read something indicating it depends on how they're exported and whether they are  a default or named export?

Reply:
Yes, exactly right. Since `logger.js` does not have a default export, but does export an object named `getLogger`, imports need to be "destructured" from the export object, which is what the braces do. In this case I think we could change this to a default export.